site/docs/guide/games.md

```diff
@@ -61,11 +61,11 @@ const keyboard = new InlineKeyboard().game("Start my_game");
 // Notice that we have used game() unlike a normal inline keyboard
 // where we use url() or text()
 
-// Via replyWithGame method
+// Via the `replyWithGame` method
 await ctx.replyWithGame("my_game", { reply_markup: keyboard });
 
-// Via api.sendGame method
-await ctx.api.sendGame(chatid, "my_game", { reply_markup: keyboard });
+// Via the `api.sendGame` method
+await ctx.api.sendGame(chatId, "my_game", { reply_markup: keyboard });
```

Comment:
Nice find.

We should change all of the other cases of `chatid` too. There is another one just above, on line 47. 😉